cmd/clusterawsadm/api/bootstrap/v1beta1/defaults.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package v1beta1
 
 import (
-	runtime "k8s.io/apimachinery/pkg/runtime"
```

Comment:
is this intentional?

Reply:
Automatic. Since the end of the import is already `runtime` there is no need to alias it to `runtime`. :)

Reply:
Cool 👍